src/server/index.test.ts

```diff
@@ -700,7 +700,7 @@ test('should include form mode when sending elicitation form requests', async ()
 
     const receivedModes: string[] = [];
     client.setRequestHandler(ElicitRequestSchema, request => {
-        receivedModes.push(request.params.mode);
+        receivedModes.push(request.params.mode ?? '');
```

Comment:
In practice, the value will never be `''` in these tests. But TypeScript insists that it it is a `string?` 🙂